### PR TITLE
[SYCL] Test for device/platform interop constructor

### PR DESCRIPTION
**Pull request by maximdimakov**
The test for https://github.com/intel/llvm/pull/3327

### CODE REVIEW

File: SYCL/Basic/device_platform_interop.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// REQUIRES: opencl
+
+// RUN: %clangxx -fsycl %s -o %t.run
+// RUN: %GPU_RUN_PLACEHOLDER %t.run
+// RUN: %CPU_RUN_PLACEHOLDER %t.run
+// RUN: %ACC_RUN_PLACEHOLDER %t.run
+
+//==--------device_platform_interop.cpp - SYCL device and platform interop
```
Comment (vladimirlaz) [inline SYCL/Basic/device_platform_interop.cpp]: line  is too long

**Review by vladimirlaz — CHANGES_REQUESTED** [review]
@maximdimakov, please fix lines lengths

File: SYCL/Basic/device_platform_interop.cpp (hunk 1 above)
Comment (maximdimakov) [reply SYCL/Basic/device_platform_interop.cpp]: Do I need to reduce length of comment or move part of it on the next line?

**Comment by vladimirlaz** [comment]
> @maximdimakov, please fix lines lengths

please shorten the line

File: SYCL/Basic/device_platform_interop.cpp (hunk 1 above)
Comment (maximdimakov) [reply SYCL/Basic/device_platform_interop.cpp]: Fixed

**Comment by maximdimakov** [comment]
@vladimirlaz Could you please review the test

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**